specs/receive_funds/02_user_interface.rst

```diff
@@ -0,0 +1,56 @@
+==========================================================
+User interface
+==========================================================
+
+=====  =============  =========  ==========
+ 68        title      tschubotz   created
+=====  =============  =========  ==========
+`68`_  Receive funds  tschubotz  2019-01-17
+=====  =============  =========  ==========
+
+.. _68: https://github.com/gnosis/safe/issues/68
+
+.. sectnum::
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+Receive funds
+-------------
+
+Receive screens that opens when e.g. tapping the
+identicon on the funds overview screen.
+
+- Tapping "Share" opens the OS share sheet.
+- Normal tap & long tap on the address copies it
+  to the clipboard.
+
+.. raw:: html
+
+   <img src="screens/android/receive funds.png" width="320px"/>
+
+.. raw:: html
+
+   <img src="screens/ios/receive funds.png" width="320px"/>
+
+
+* Zeplin link Android https://zpl.io/am6Z9wm
+* Zeplin link iOS https://zpl.io/boKADwg
+
+Receive funds - Copied address
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Toast / indicationj that an address was copied.
+
+.. raw:: html
+
+   <img src="screens/android/receive funds - copied.png" width="320px"/>
+
+.. raw:: html
+
+   <img src="screens/ios/receive funds - copied.png" width="320px"/>
+
+
+* Zeplin link Android https://zpl.io/am6Z9wm
```

Comment:
This link should be https://zpl.io/aByP18A

Reply:
Thank you 🙈